modules/dynamic-plugins-get-started.adoc

```diff
@@ -0,0 +1,108 @@
+= Getting started with dynamic plugins
+
+There are different customizations you can make to the {product-title} web console. The following procedure sets up your environment to write a new OpenShift Console dynamic plugin, and adds a tab to the *Pod details* page as an example extension to your plugin.
+
+[NOTE]
+====
+The {product-title} web console runs in a container connected to the cluster you have logged into. See "Running your dynamic plugin" for information to test the plugin before creating your own.
+====
+
+.Prerequisites
+* Ensure you have link:https://nodejs.org/en/[`Node.js`] installed.
+* Ensure you have link:https://yarnpkg.com/[`yarn`] installed.
+
+.Procedure
+
+.Procedure
+
+. Visit the link:https://github.com/openshift/console-plugin-template[`console-plugin-template`] repository containing a template for creating plugins in a new tab.
++
+[IMPORTANT]
+====
+Custom plugin code is not supported by Red Hat. Only link:https://access.redhat.com/solutions/5893251[Cooperative community support] is available for your plugin.
+====
+
+. Select the *Use this template* dropdown button and select *_Create new repository_* from the dropdown list to create a GitHub repository.
+
+. Re-name the new repository with the name of your plugin.
+
+. Clone your copied repository to your local machine so you can edit the code.
+
+. Edit the plugin metadata in the `consolePlugin` declaration of `package.json`.
++
+[source,json]
+----
+"consolePlugin": {
+  "name": "my-plugin", <1>
+  "version": "0.0.1", <2>
+  "displayName": "My Plugin", <3>
+  "description": "Enjoy this shiny, new console plugin!", <4>
+  "exposedModules": {
+    "ExamplePage": "./components/ExamplePage"
+  },
+  "dependencies": {
+    "@console/pluginAPI": "/*"
+  }
+}
+----
+<1> Update the name of your plugin.
+<2> Update the version.
+<3> Update the display name for your plugin.
+<4> Update the description with a synopsis about your plugin.
+
+. Add the following to the `console-extensions.json` file:
++
+[source,json]
+
+----
+{
+  "type": "console.tab/horizontalNav",
+  "properties": {
+    "page": {
+      "name": "Example Tab",
+      "href": "example"
+    },
+    "model": {
+      "group": "core",
+      "version": "v1",
+      "kind": "Pod"
+    },
+    "component": { "$codeRef": "ExampleTab" }
+  }
+}
+----
+
+. Edit the `package.json` file to include the following changes:
++
+[source,json]
+
+----
+        "exposedModules": {
+            "ExamplePage": "./components/ExamplePage",
+            "ExampleTab": "./components/ExampleTab"
+        }
+----
+
+. Write a message to display on a new custom tab on the *Pods* page by creating a new file `src/components/ExampleTab.tsx` and adding the following script:
++
+[source,tsx]
+
+----
+import * as React from 'react';
+
+export default function ExampleTab() {
+    return (
+        <p>This is a custom tab added to a resource using a dynamic plugin.</p>
+    );
+}
+----
+
+.Verification
```

Comment:
Shall we also mention that user should deploy && enable the plugin before visiting a Pod page to view the added tab?

Reply:
Yes, thank you! I have added the step with a reference to the doc for further reading.